CHANGELOG.md

```diff
@@ -9,28 +9,38 @@ Ensure you use consistent title format.
 
 **Unreleased**
 
+!!! note "Multiple groups of instances and groups of users"
+
+    temBoard 9.0 is planned to remove the ability to put an instance in several groups of instances
+    and a group of instances in several groups of users.
+
+    temboard 8.2 warn's you about instances and groups of instances matching these cases.
```

Comment:
`    temboard 8.2 warns you about instances and groups of instances matching these cases.`